src/info/mod.rs

```diff
@@ -408,27 +409,16 @@ mod tests {
         let info_str = format!("{}", info);
         let info_u8 = strip_ansi_escapes::strip(&info_str)?;
         let simple_info_str = std::str::from_utf8(&info_u8)?;
-        let expected_info = [
-            r"(?s)onefetch-committer-name ~ git version .+",
-            r"-{37,}",
-            r"Project: repo",
-            r"HEAD: .+ \(main\)",
-            r"Created: 22 years ago",
-            r"Language:",
-            r".+Verilog \(100.0 %\).+",
-            r"Author: 100% author 2",
-            r"Last change: 22 years ago",
-            r"Repo: https://github.com/user/repo.git",
-            r"Commits: 2",
-            r"Lines of code: 1",
-            r"Size: 6 B \(1 file\)",
-        ]
-        .join(".");
-        let re = Regex::new(&expected_info).unwrap();
+        #[cfg(windows)]
+        let expected_regex = include_str!("..\\..\\tests\\regex\\test_verilog_repo.stdout.regex");
+        #[cfg(unix)]
```

Comment:
```suggestion
```
I'm pretty sure the path will be resolved correctly on Windows.

Reply:
ok, will give it a try, I read that [the include_str path was operating system dependent](https://doc.rust-lang.org/std/macro.include_str.html#). I see there is a CI test on windows, I don't have any ability to test that, all OSX or linux on my side.

Reply:
Yeah TBH those docs seem a bit misleading (maybe worth a PR? :wink:), but I think it's just a warning that Windows path separators will *only* work on Windows. I think modern Windows OSs always treat `/` just like `\` in paths (at least that happens when I accidentally use the wrong separator on Windows :lauging:). Additionally, AFAIK most modern programming languages will treat `/` as OS-independent.